css/style.css

```diff
@@ -1,281 +1,230 @@
 * {
-	box-sizing:border-box;
-	font-family:arcadeclassic;
-	color:white;
-	cursor:url("../images-logo/cursor-spaceship-medium.png"),auto;
+	box-sizing: border-box;
+	font-family: arcadeclassic;
+	color: #FFFFFF;
+	cursor: url("../images-logo/cursor-spaceship-medium.png"),auto;
 }
 
 @font-face {
-  font-family: arcadeclassic;
-  src: url("../arcadeclassic/arcadeclassic.ttf");
+	font-family: arcadeclassic;
+	src: url("../arcadeclassic/arcadeclassic.ttf");
 }
 
 body {
-	
-	background-image:url("../images-logo/planet2.png");
+	background-image: url("../images-logo/planet2.png");
 	background-size: 2200px 1122px;
-	background-repeat: no-repeat;
+	background-repeat: repeat;
 	background-blend-mode: multiply;
-	
 }
 
 .current {
-	border-top:2px solid white;
-	border-bottom:2px solid white;
+	border-top: 2px solid #FFFFFF;
+	border-bottom: 2px solid #FFFFFF;
 }
 
 .container {
-	display:flex;
-	justify-content:space-between;
-	width:80%;
-	margin-left: auto;
-	margin-right:auto;
+	display: flex;
+	justify-content: space-between;
+	width: 80%;
+	margin: 0 auto;
 }
 
 .link {
-	display:flex;
-	justify-content:flex-end;
+	display: flex;
+	justify-content: flex-end;
 }
 
-/*------------------Navbar start here------------------*/
-
 .navbar {
 	margin-top: 20px;
-	background-color:none;
 }
+
 .navbar h1 {
-	border-left:3px solid white;
+	border-left: 3px solid #FFFFFF;
 	text-align: right;
 	width: 5rem;
-	padding:10px;
-	margin-right:20px;
+	padding: 10px;
+	margin-right: 20px;
 }
 
 .navlink {
-	display:inline-block;
-	margin-top:auto;
-	margin-bottom:auto;
-	margin-right:10px;
-	padding-top:5px;
+	display: inline-block;
+	margin: auto auto;
+	padding-top: 5px;
 	width: 6rem;
 	height: 2rem;
-	font-size:1rem;
-	text-align:center;
-	text-decoration:none;
-	
+	font-size: 1rem;
+	text-align: center;
+	text-decoration: none;
 }
 
-a:active{
-	border-top:2px solid white;
+a:active {
+	border-top: 2px solid #FFFFFF;
 }
 
 a::after {
-	content:'';
-	display:block;
-	width:0;
-	height:2px;
-	background-color:white;
+	content: '';
+	display: block;
+	width: 0;
+	height: 2px;
+	background-color: #FFFFFF;
 	transition: width .4s
 }
 
 a:hover::after {
-	width:100%;
+	width: 100%;
 	transition: width .4s;
 }
-/*------------------Body Main start here------------------*/
+
 main {
-	position:relative;
+	position: relative;
+	height: auto;
+	
 }
 
 .content-box {
-	width:29rem;
-	margin:auto auto;
-	padding-top:10px;
-    text-align: center;
+	width: 29rem;
+	margin: auto auto;
+	padding-top: 10px;
+	text-align: center;
     text-justify: inter-word;
-	letter-spacing:0.11rem;
-	font-size:1.4rem;
+	letter-spacing: 0.11rem;
+	font-size: 1.4rem;
 }
 /*-----------------------HOME / INDEX SECTION---------------------*/
-
 .border {
 	border: 10px solid transparent;
 	border-image: url("../images-logo/border2.png") 10% stretch;	
+	background-color: #00000090;
```

Comment:
use rgba for transparency, 8 digit hex doesn't work in all browsers:
https://caniuse.com/css-rrggbbaa